sharktank/sharktank/utils/debugging.py

```diff
@@ -144,7 +144,7 @@ def trace_tensor_to_safetensors_callback(key: str, *tensors: Tuple[torch.Tensor]
 
 def trace_tensors_to_safetensors(key: str, tensors: Dict[str, torch.Tensor]):
     # Sanitize as path.
-    key = re.sub("[" + re.escape(r"""#~!@$%^&*()[]{}:;"'""") + "]", "", key)
```

Comment:
seems like we want to escape some more characters, would it be better to just add new chars to the existing sequence?
something like this `"[" + re.escape(r"""#~!@$%^&*()[]{}:;"'"*?"<>|""") + "]"`

assuming the existing pattern is useful for some cases atleast.

Reply:
I don't want to escape all these characters. They are legal file name characters on all platforms so they should be OK. In particular I don't want `%` to be escaped.